gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "2.13.3",
+  "version": "2.13.4",
```

Comment:
Probably should be a minor version since it's a new capability.

Reply:
> Probably should be a minor version since it's a new capability.

Ok: https://github.com/sonatype/sonatype-react-shared-components/pull/231/commits/67bc53e2af8fa4d64ea0ef5a5c5c4b9c98ea5c77